recipes/rmlui/4.x/conanfile.py

```diff
@@ -85,78 +86,96 @@ def requirements(self):
             self.requires("lua/5.3.5")
 
         if self.options.with_thirdparty_containers:
-            self.requires("robin-hood-hashing/3.11.3")
+            self.requires("robin-hood-hashing/3.11.3", headers=True, transitive_headers=True)
```

Comment:
Headers is true but default

```suggestion
            self.requires("robin-hood-hashing/3.11.3", transitive_headers=True)
```